.github/workflows/wheels.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
     env:
       CIBW_SKIP: "pp*"
-      CIBW_BEFORE_BUILD: pip install "numpy>=1.26.2" --config-settings=setup-args="-Dallow-noblas=true"
+      CIBW_BEFORE_BUILD: pip install "numpy>=2.0.0" --config-settings=setup-args="-Dallow-noblas=true"
```

Comment:
@phofl I'm not very familiar with the wheel building infrastructure here. Just wanted to confirm that with this users should still be able to install and use `crick` with `numpy=1.x`

Reply:
Yes, that's my understanding of https://numpy.org/devdocs/dev/depending_on_numpy.html#numpy-2-0-specific-advice

Reply:
Thanks for confirming and pointing to those docs